src/doc/fxhash/redeemables.md

```diff
@@ -8,15 +8,15 @@ description: "Redeemables provide a framework to connect NFTs with their physica
 
 # What are redeemables ?
 
-Redeemables were introduced to facilitate and secure the necessary steps artists have to go through when they want to provide a physical counterpart to their digital projects. In essence, redeemables are an extension which can be plugged to any existing project, allowing such project to become redeemable.
+Redeemables were introduced to facilitate and secure the necessary steps artists have to go through when they want to provide a physical counterpart to their digital projects. In essence, redeemables are an extension that can be attached to any existing project, allowing that project to become redeemable.
 
-Redeemables provide a set of modules allowing the platform to request arbitrary data, which gets signed and is associated to a redeem event when it occurs on-chain. It allows us to map on-chain redemptions with the data signed and collected off-chain, which in turns enables various physical events tooccur, mainly shipping a product associated with a NFT.
+Redeemables provide a set of modules allowing the platform to request arbitrary data. This data is then signed and is associated with a redeem event when it occurs on-chain. It enables us to map on-chain redemptions with the data signed and collected off-chain, facilitating various physical events, primarily the shipping of a product associated with an NFT.
 
-Redemptions are non-destructive. When you redeem your NFT, it still exists after the redemption and you keep its ownership.
+Redemptions are non-destructive. When you redeem your NFT, it continues to exist after the redemption and you keep its ownership.
 
-# Is the feature opened ?
+# Is the feature open ?
 
-Redeemables are rather complex, and for now can only be created from our admin dashboard. Before releasing the feature to the public, we want to carefully test it and make sure we cover the basic use-cases. We are opened to receiving requests if you want to experiment with redeemables.
```

Comment:
Redeemables are rather complex and, for now, can only be created via our internal admin dashboard. Before releasing the feature to the public, we want to test it carefully and ensure we cover the basic use cases. We are open to receiving requests if you want to experiment with redeemables.